plugins/urls.py

```diff
@@ -97,6 +97,11 @@ def hackernews(self, msg):
         fmt = "{by}: «{title}» {url}"
         msg.reply(fmt.format(**post))
 
+    # @BotPlugin.command(r'.*https?://(www\.)?youtube\.com/watch\?v=(\w+)' + end_url)
```

Comment:
![Troll?](http://www.quickmeme.com/img/79/79f0be6c87e66a2fd0ddcb521661a1299b5f029f9f4c80a88eafc5a1209faa66.jpg)

Reply:
why not both ?